python/ray/data/tests/test_dataset.py

```diff
@@ -4941,19 +4941,16 @@ def check_dataset_is_local(ds):
     # Plain read.
     ds = ray.data.read_parquet(local_path).fully_executed()
     check_dataset_is_local(ds)
-    assert "1 nodes used" in ds.stats(), ds.stats()
```

Comment:
@jianoaix - this assertion because flaky due to the async stats actor metadata fetching - https://github.com/ray-project/ray/pull/29635#discussion_r1018409717 . Given `check_dataset_is_local(ds)` already served the test purpose. It'd better to avoid flakiness here. We need deprecate stats actor soon..